compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -2272,14 +2272,14 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         def fullName: String = self.denot.fullName.toString
         def pos: Position = self.sourcePos
 
-        def documentation: Option[Documentation] =
+        def docstring: Option[String] =
```

Comment:
```suggestion
        def rawDocumentation: Option[String] =
```

Reply:
Should add documentation to this method.

Reply:
We do have docs in `trait Quotes`, which is where we actually care about docs, right?

Reply:
Ignore that, I was looking in the wrong file